data/json/items/gun/flintlock.json

```diff
@@ -14,8 +14,8 @@
       "ranged_damage": -4,
       "dispersion": 60,
       "recoil": 15
-    },
-    "proportional": { "reload": 0.5 }
+      "reload": -400
```

Comment:
That is the JSON error I encountered and have now fixed, as I stated in the main discussion.